repl/scala-2.11/src/main/scala/org/apache/spark/repl/SparkILoop.scala

```diff
@@ -7,82 +7,44 @@ package scala
 package tools.nsc
```

Comment:
Can we change the package org.apache.spark.repl ? Is there something package private we are dealing with ?